assets/styles/jon.scss

```diff
@@ -0,0 +1,8 @@
+#test{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space